packages/fxa-auth-server/config/index.ts

```diff
@@ -794,6 +794,14 @@ const conf = convict({
       env: 'SUBSCRIPTIONS_ENABLED',
       default: false,
     },
+    paymentsServer: {
+      url: {
+        doc: 'The url of the corresponding fxa-payments-server instance',
+        env: 'PAYMENTS_SERVER_URL',
+        format: 'url',
+        default: 'https://subscriptions.firefox.com',
```

Comment:
We should mention this new config in the deploy doc.  Ops do need to set a value for dev (https://latest.dev.lcip.org/) and stage.

Reply:
Thanks for bringing this up; I/you added a [note](https://docs.google.com/document/d/1lc5T1ZvQZlhXY6j1l_VMeQT9rs1mN7yYIcHbRPR2IbQ/edit#heading=h.okr2i14td32j) for Train 204 in the deploy docs.